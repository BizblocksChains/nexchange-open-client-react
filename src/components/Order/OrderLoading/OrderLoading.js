import React from 'react';
import { I18n } from 'react-i18next';

const OrderLoading = () => {
  return (
    <I18n ns="translations">
      {t => (
        <div className="text-center">
          <h2>{t('loading')}...</h2>
        </div>
      )}
    </I18n>
  );
};

export default OrderLoading;
