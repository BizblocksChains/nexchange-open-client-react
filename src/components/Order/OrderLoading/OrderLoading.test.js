import React from 'react';
import { shallow } from 'enzyme';
import OrderLoading from './OrderLoading';

describe('OrderLoading', () => {
  it('renders correctly', () => {
    const wrapShallow = shallow(<OrderLoading />).dive();
    expect(wrapShallow).toMatchSnapshot();
  });
});
