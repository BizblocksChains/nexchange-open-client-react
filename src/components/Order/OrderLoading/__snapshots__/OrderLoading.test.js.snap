// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderLoading renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <I18n
    ns="translations"
>
    [Function]
</I18n>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h2>
        loading
        ...
</h2>,
      "className": "text-center",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "loading",
          "...",
        ],
      },
      "ref": null,
      "rendered": Array [
        "loading",
        "...",
      ],
      "type": "h2",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h2>
          loading
          ...
</h2>,
        "className": "text-center",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "loading",
            "...",
          ],
        },
        "ref": null,
        "rendered": Array [
          "loading",
          "...",
        ],
        "type": "h2",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
